services/blockchain-indexer/shared/dataService/business/pendingTransactions.js

```diff
@@ -61,7 +59,7 @@ const getPendingTransactionsFromCore = async () => {
 				};
 			}
 
-			updateAccountPublicKey(normalizedTransaction.senderPublicKey);
+			indexAccountByPublicKey(normalizedTransaction.senderPublicKey);
```

Comment:
Since this queue ultimately only indexes the publicKey and address, renaming the method to `updateAccountPublicKey` or `indexAccountPublicKey` would make more sense.